index.html

```diff
@@ -0,0 +1,286 @@
+                </div>
+              </div>
+            </div>
+            </div>
+          </section>
+          <section>  
+          <div class="Portfolio__About__MySelf">
+            <h4 class="About__MySelf__header">About Myself</h4>
+            <div class="About__MySelf__Content">
+              <p>Hello I’m a software developer! I can help you build a product , feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
+            </div>
+            <div class="About__MySelf__layout">
+              <div class="About__MySelf__design">
+                <div class="About__MySelf__Connect">LET'S CONNECT</div>
+              </div>
+              <div class="social-media">
+              <ul>
+                <li><a href=""><img src="img/t_icon.png" alt="Twitter icon" /></a></li>
+                <li><a href=""><img src="img/Li_icon.png" alt="LinkedIn icon" /></a></li>
+                <li><a href=""><img src="img/m_icon.png" alt="Medium icon" /></a></li>
+                <li><a href=""><img src="img/_gth_icon.png" alt="Github icon" /></a></li>
+                <li><a href=""><img src="img/finger_c.png" alt="finger icon" /></a></li>
+            
+            </ul>
+           </div>
+           <div class="resume__tag">
+            <button class="resume__button"><p>Get my resume</p></button>
+           </div>
+        <div>
+          <ul>
+            <li class="Languages">
+              <h3>Languages</h3>
+               <img class="framework__arr" src="img/about_page/arr_d.svg"></div>
+                <ul>
+                   <li><div class="resume__coding">
+                     <img src="img/about_page/javascript.png" alt="">
+                     <p class="resume__coding_par">JavaScript</p></div>
+                   </li>
+                   <li><div class="resume__coding">
+                      <img src="img/about_page/html.png" alt="">
+                      <p class="resume__coding_par">HTML</p></div>
+                    </li>
+                    <li><div class="resume__coding">
+                       <img src="img/about_page/css.png" alt="">
+                       <p class="resume__coding_par">CSS</p></div>
+                    </li>
+                 </ul>
+             </li>
+            <li class="framework__container">   
+              <ul>
+                <li>
+                   <p class="Portfolio_About_d_1">Frameworks</p> 
+                   <img class="framework__arr" src="img/about_page/arr_r.svg">
+                </li>
+              </ul>
+            <li class="framework__container">
+              <ul>
+                <li>
+                  <p class="Portfolio_About_d_1">Skills</p>
+                  <img class="framework__arr" src="img/about_page/arr_r.svg">
+                </li>
+              </ul>
+            </li>
```

Comment:
- [ ] Kindly add the underline to your design so as to match the given Figma template

Your design
![image](https://github.com/AsciencioAlex/Mobile-Finish-Portfolio/assets/81932478/d09eeb0d-24dd-476b-b003-fc9d341b33b7)

Given Figma template
![image](https://github.com/AsciencioAlex/Mobile-Finish-Portfolio/assets/81932478/a0bec363-bb07-4bee-9d14-9e8849484617)